### DISCUSSION TITLE
How to specify default admin username for Keycloak devservices?

### DISCUSSION THREAD

**Discussion by Eng-Fouad** [quarkusio/quarkus#32148] [Q&A] [answered]
The default username is `admin` and default password is `admin`. Is it possible to change that? I am using:

    %dev.quarkus.keycloak.devservices.image-name=quay.io/keycloak/keycloak:21.0

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment, marked as answer]
@Eng-Fouad This password is only used for a DevServices Keycloak instance, why would you like to customize it ?
